packages/cache/src/CacheStore.spec.tsx

```diff
@@ -81,6 +87,19 @@ describe('CacheStore', () => {
     expect(cacheStore.getError(errorCache(2))).toBeUndefined()
   })
 
+  it('should take a key and remove the cached data for that key', async () => {
+    try {
+      cacheStore.suspend(successCache(1))
```

Comment:
@manudeli I'm curious about what kind of operation you had in mind for the try method.

Reply:
I just considered just re-naming suspend as try or add or get. in my opinion, cacheStore.suspend is weird for me.

Reply:
@manudeli I also think it is necessary to change to explicit naming. As an alternative, how about `cacheStore.fetch`?

Reply:
I also like fetchStore.get. However, I think 'try' is too abstract to represent a method, and 'add' does not sufficiently explain the content of the method. Additionally, since the name is @suspensive/cache, users will naturally understand that uncached data will be handled by the cache.

Reply:
Wow... fetchCacheStore.get(fetchCache(1)) cool

Reply:
CacheStore seems more explicit compared to fetchCacheStore. Users might find it confusing with fetchCacheStore. What about `cacheStore.get(cache(1))`?